redisgears_v8_plugin/src/v8_backend.rs

```diff
@@ -52,15 +52,30 @@ unsafe impl GlobalAlloc for Globals {
 static mut GLOBAL: Globals = Globals { backend_ctx: None };
 
 pub(crate) fn log(msg: &str) {
-    unsafe { (GLOBAL.backend_ctx.as_ref().unwrap().log)(msg) };
+    #[cfg(not(feature = "test"))]
+    unsafe {
+        (GLOBAL.backend_ctx.as_ref().unwrap().log)(msg)
+    };
+    #[cfg(feature = "test")]
+    println!("log message: {msg}");
 }
 
 pub(crate) fn get_fatal_failure_policy() -> LibraryFatalFailurePolicy {
-    unsafe { (GLOBAL.backend_ctx.as_ref().unwrap().get_on_oom_policy)() }
+    #[cfg(not(feature = "test"))]
+    unsafe {
+        (GLOBAL.backend_ctx.as_ref().unwrap().get_on_oom_policy)()
+    }
+    #[cfg(feature = "test")]
+    ..Default::default()
```

Comment:
Can we return Abort here instead of implement default? I am afraid that by implement default we will accidentally take default value instead of the value set on the configuration.